playbook/workshops/contrib/ajar/README.md

```diff
@@ -234,4 +234,179 @@ You've finished Part 1!
 
 ![](img/celebrate.gif)
 
-# More coming soon...
+
+## Part 2
+
+Welcome to part 2 of the Ajar.io workshop!
+
+To recap, in [part 1](#part-1) you learned about:
+
+- **JavaScript variables**: used to store data values. Declared using the `var`
+  keyword. You can make variables like this:
+
+```js
+var x = 10 * 2;
+var name = "Bogdan";
```

Comment:
It might be helpful to remind them they can retrieve a value from the variable by calling it.

ex:
```js
var x = 10 * 2;
var name = "Bogdan";

name;
// => Bogdan
```